src/main/java/com/radixdlt/atommodel/Atom.java

```diff
@@ -248,42 +245,27 @@ private void setSignature(EUID id, ECDSASignature signature) {
 		this.signatures.put(id, signature);
 	}
 
-	public void sign(ECKeyPair key) throws AtomAlreadySignedException {
-		if (!this.signatures.isEmpty()) {
+	public static void sign(Atom atom, ECKeyPair key, Hasher hasher) throws AtomAlreadySignedException {
```

Comment:
Why is this a static method now?

Reply:
it needs a Hasher for signing

Reply:
missed your point. yeah, they don't need to be static. switched back to regular methods